packages/account-fs/agent/runtime.js

```diff
@@ -4,19 +4,24 @@ import { spki } from 'iso-signatures/spki'
 import { CID } from 'multiformats/cid'
 import * as uint8arrays from 'uint8arrays';
 import localforage from "localforage";
+import { createClient } from 'redis';
 
 const SHOVEL_FS_ACCESS_KEY = "SHOVEL_FS_ACCESS_KEY"
 const SHOVEL_FS_FOREST_CID = "SHOVEL_FS_FOREST_CID"
 const SHOVEL_AGENT_WRITE_KEYPAIR = "SHOVEL_AGENT_WRITE_KEYPAIR"
+const SHOVEL_AGENT_DID = "SHOVEL_AGENT_DID"
```

Comment:
We don't need to introduce this new value.
it can be computed from `SHOVEL_AGENT_WRITE_KEYPAIR`, via `signer` method as
```
    const signer = await this.signer()
    return signer.did
```